recipes/sensv/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "sensv" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/HKU-BAL/SENSV/archive/v{{ version }}.zip
+  sha256: e8544c106a3d88a399ec63ea829b9b03497e171814f175f54af601e5f655a9c3
+
+build:
+  number: 0
+  skip: True  # [osx]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - zlib
+  run:
+    - zlib
+    - python >=3.7, <3.8
+    - minimap2=2.17
```

Comment:
Do you need these exact versions of minimap2 et al.?

Reply:
Yes I think I need these. :)